cypress/integration/pages/onDemandRadioEpisode/tests.js

```diff
@@ -0,0 +1,64 @@
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
+  describe(`Tests for ${service} ${pageType}`, () => {
+    describe('On Demand Radio body', () => {
+      it('should render a H1, which contains/displays a styled headline', () => {
+        cy.request(`${Cypress.env('currentPath')}.json?renderer_env=live`).then(
+          ({ body }) => {
+            cy.get('h1').should('contain', body.promo.headlines.headline);
+          },
+        );
+      });
+
+      it('should render a paragraph, which contains/displays a styled summary', () => {
+        cy.request(`${Cypress.env('currentPath')}.json?renderer_env=live`).then(
+          ({ body }) => {
+            cy.get('[role="main"] p').should(
+              'contain',
+              body.promo.media.synopses.medium,
+            );
+          },
+        );
+      });
+    });
+
+    describe('Brand image visible above 400, not visible below 400', () => {
+      const sizesBelowBreakpoint = ['iphone-6'];
+
+      sizesBelowBreakpoint.forEach(size => {
+        // make assertions on the image using
+        // an array of different viewports
+        it(`Should display image on ${size} screen`, () => {
+          if (Cypress._.isArray(size)) {
+            cy.viewport(size[0], size[1]);
+          } else {
```

Comment:
This is probably unnecessary if we decide to use the named devices for this test: https://docs.cypress.io/api/commands/viewport.html#Arguments